pkg/agent/secondarynetwork/cnipodcache/cache.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Antrea Authors
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http:// www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cnipodcache
+
+import (
+	"sync"
+
+	"k8s.io/client-go/tools/cache"
+
+	"antrea.io/antrea/pkg/agent/util"
+	"antrea.io/antrea/pkg/util/k8s"
+)
+
+const (
+	podIndex = "pod"
+)
+
+type CNIPodInfoCache struct {
+	sync.RWMutex
+	cache cache.Indexer
+}
+
+// Add CNIPodInfo to local cache store.
+func (pod *CNIPodInfoCache) AddCNIConfigInfo(CNIConfig *CNIConfigInfo) {
```

Comment:
But why you use "pod" for short name of CNIPodInfoCache, but not "cache" or just "c"?

Reply:
CNIConfig -> cniConfig (as it is not a global variable).